Description: Drop support of bndlib Spring Framework support
 since this library is not build with this support enabled.
Author: Damien Raude-Morvan <drazzib@debian.org>
Forwarded: not-needed
Last-Update: 2011-10-13

--- a/src/main/java/org/apache/felix/bundleplugin/BundlePlugin.java
+++ b/src/main/java/org/apache/felix/bundleplugin/BundlePlugin.java
@@ -107,7 +107,6 @@
 import aQute.bnd.osgi.Packages;
 import aQute.bnd.osgi.Processor;
 import aQute.lib.collections.ExtList;
-import aQute.lib.spring.SpringXMLType;
 import aQute.libg.generics.Create;
 
 
@@ -1839,10 +1838,6 @@
 
         properties.put( "classifier", classifier == null ? "" : classifier );
 
-        // Add default plugins
-        header( properties, Analyzer.PLUGIN, BlueprintPlugin.class.getName() + ","
-                                           + SpringXMLType.class.getName() + ","
-                                           + JpaPlugin.class.getName() );
 
         return properties;
     }
